src/components/AudioControls/AudioControls.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+import AudioControls from './AudioControls'
+  
+describe('AudioControls', () => {
+    it('renders', () => {
+        render(<AudioControls />)
+        expect(screen.getByText('AudioControls')).toBeInTheDocument()
+    })
+})
+      
```

Comment:
you could add some simple tests if a button etc is rendered

Reply:
Yes i will do a full test on another day.